openedx/core/djangoapps/oauth_dispatch/docs/decisions/0014-add-grant-type-in-jwt-payload.rst

```diff
@@ -0,0 +1,30 @@
+14. Added grant type in JWT payload
+-----------------------------------------
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+Open edX mobile apps are `migrating to JWT token`_ authentication.Therefore gradually adding JWT support in required APIS.
+As part of the current mobile authentication flow, we exchange an opaque access token for a session.
+This is only allowed for access tokens created with the password grant type.
+However, as we migrate from using the opaque access token to a JWT, we are no longer able to determine the grant type of the JWT.
+
+.. _migrating from Dot access token to JWT token: 0013-mobile-migration-to jwt.rst
```

Comment:
1. I'm still looking for this ADR. Did you think it was added?
2. The label needs to change to `migrating to JWT token`.
3. The URL needs to be fixed. The filename has a space in it, and I'm not sure if the filename only will work.